wellcomeml/viz/visualize_clusters.py

```diff
@@ -0,0 +1,31 @@
+from bokeh.plotting import figure, show
+from wellcomeml.viz.palettes import Wellcome33
+from bokeh.io import output_notebook
+output_notebook()
+
+
+def visualize_clusters(reduced_points: list):
+
```

Comment:
Remove extra line

Reply:
👍  done

Reply:
@aCampello flake8 actually requires this extra line